opentelemetry-stdout/Cargo.toml

```diff
@@ -28,12 +28,12 @@ logs = ["opentelemetry/logs", "opentelemetry_sdk/logs", "opentelemetry_sdk/spec_
 
 [dependencies]
 chrono = { workspace = true, features = ["now"] }
-opentelemetry = { version = "0.30", path = "../opentelemetry" }
-opentelemetry_sdk = { version = "0.30", path = "../opentelemetry-sdk" }
+opentelemetry = { workspace = true }
+opentelemetry_sdk = { workspace = true }
 
 [dev-dependencies]
-opentelemetry = { path = "../opentelemetry", features = ["metrics"] }
-opentelemetry_sdk = { path = "../opentelemetry-sdk", features = ["metrics"] }
+opentelemetry = { workspace = true, features = ["metrics"] }
+opentelemetry_sdk = { workspace = true, features = ["metrics"] }
```

Comment:
These two packages are already available as regular dependencies, re-declaring them in dev-dependencies seems unnecessary. Though not coming from this PR.